plugins/search-backend-module-catalog/src/alpha.ts

```diff
@@ -23,24 +23,24 @@ import {
   coreServices,
   createBackendModule,
 } from '@backstage/backend-plugin-api';
-import { TaskScheduleDefinition } from '@backstage/backend-tasks';
-import { searchIndexRegistryExtensionPoint } from '@backstage/plugin-search-backend-node/alpha';
-
+import { catalogServiceRef } from '@backstage/plugin-catalog-node/alpha';
 import {
+  CatalogCollatorEntityTransformer,
   DefaultCatalogCollatorFactory,
-  DefaultCatalogCollatorFactoryOptions,
 } from '@backstage/plugin-search-backend-module-catalog';
-import { catalogServiceRef } from '@backstage/plugin-catalog-node/alpha';
+import { searchIndexRegistryExtensionPoint } from '@backstage/plugin-search-backend-node/alpha';
+import { readScheduleConfigOptions } from './collators/config';
 
 /**
- * @alpha
  * Options for {@link searchModuleCatalogCollator}.
+ *
+ * @alpha
  */
-export type SearchModuleCatalogCollatorOptions = Omit<
-  DefaultCatalogCollatorFactoryOptions,
-  'discovery' | 'tokenManager' | 'catalogClient'
-> & {
-  schedule?: TaskScheduleDefinition;
+export type SearchModuleCatalogCollatorOptions = {
+  /**
+   * Allows you to customize how entities are shaped into documents.
+   */
+  entityTransformer?: CatalogCollatorEntityTransformer;
 };
```

Comment:
I'm wondering why you add this option here. If I understand the [documentation](https://backstage.io/docs/features/search/how-to-guides#how-to-customize-fields-in-the-software-catalog-index) correctly there is an alternative approach to `searchModuleCatalogCollator(...)` for adding the catalog collator. 

Did that changed in the new backend system? Is this approach easier?